<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBoxChannel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>L1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ON/OFF</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxON">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderPower">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>255</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1023</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPower">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1023</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIPAddress">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>IP address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
